### PR TITLE
[typo] fix typo in comments & const value(SignalUsr2)

### PR DESCRIPTION
**Pull request by jxd134**

### CODE REVIEW

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 180 Deploy Test Success

**Comment by benja-wu** [comment]
I notice the `pkg/util/httpfilter/httpfilter.go` modification appears again. 
Doesn't it has been merged already? I think you can begin a new branch and cheery-pick the desired commit, which is `53bd6e3`.

**Review by megaeasex — APPROVED** [review]
[TASK:easegress-pr-test SUCCESS]megaease/easegress Pull Request 180 Deploy Test Success

File: pkg/context/httpcontext.go [hunk 1]
```diff
@@ -317,7 +317,7 @@ func (ctx *httpContext) Log() string {
 		strings.Join(ctx.tags, " | "))
 }
 
-// Template returns HTTPTemplate rely interface
+// Template returns HTTPTemplate rely on interface
```
Comment (localvar) [inline pkg/context/httpcontext.go:320]:
```suggestion
// Template returns the template engine
```

**Pull request closed without merging**